<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportActionsByServiceTypeSetupDialog</class>
 <widget class="QDialog" name="ReportActionsByServiceTypeSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkPodtver">
     <property name="text">
      <string>Подтверждение оплаты</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblPodtver">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Тип подтверждения:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="cmbPodtver">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>не выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>оплачено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>отказано</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblBegDatePodtver">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Начало периода подтверждения:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="CDateEdit" name="edtBegDatePodtver">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblEndDatePodtver">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Окончание периода подтверждения:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="CDateEdit" name="edtEndDatePodtver">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>chkPodtver</tabstop>
  <tabstop>cmbPodtver</tabstop>
  <tabstop>edtBegDatePodtver</tabstop>
  <tabstop>edtEndDatePodtver</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportActionsByServiceTypeSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportActionsByServiceTypeSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPodtver</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbPodtver</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPodtver</sender>
   <signal>clicked(bool)</signal>
   <receiver>edtBegDatePodtver</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPodtver</sender>
   <signal>clicked(bool)</signal>
   <receiver>edtEndDatePodtver</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
